Makefile

```diff
@@ -8,14 +8,14 @@ include CREDENTIALS
 
 .PHONY: build_only
 build_only:
-	-rm ./build/*.xpi
-	web-ext build --source-dir=src --artifacts-dir=build
-	mv build/wayback_machine-$(BUILD_VERSION).xpi build/Wayback_Machine_Firefox_V$(BUILD_VERSION).xpi
-	git add *.xpi
+	-rm ./build/*.zip
+	npm run build_only
```

Comment:
This is the tricky bit here. By calling `npm run <task>`, npm will correctly resolve the locally installed module in ./node_modules/.bin/ (because npm adds that to your `$PATH` automatically for you) instead of expecting you to have the module installed globally.